docusaurus.config.ts

```diff
@@ -176,7 +152,7 @@ const config: Config = {
           items: [
             {
               label: 'Installation',
-              to: '/docs/getting-started/prerequisites',
+              to: '/docs/category/getting-started',
             }
           ],
```

Comment:
LOGAF
Why not add more items here while you're at it?
I feel that our footer is quite "empty".

Reply:
I just didn't bother with it. We could add all the community links there but it's a little outside of scope tbh I just moved and fixed spelling mistakes or broken links